### PR TITLE
AF-822: Fix null in error message when uploading non-existing file

### PR DESCRIPTION
**Pull request by tomasdavidorg**
https://github.com/kiegroup/kie-wb-common/pull/1282

### CODE REVIEW

File: uberfire-extensions/uberfire-widgets/uberfire-widgets-core/uberfire-widgets-core-client/src/test/java/org/uberfire/ext/widgets/core/client/editors/defaulteditor/UploadEventTest.java [hunk 1]
```diff
@@ -0,0 +1,39 @@
+package org.uberfire.ext.widgets.core.client.editors.defaulteditor;
```
Comment (manstis) [inline uberfire-extensions/uberfire-widgets/uberfire-widgets-core/uberfire-widgets-core-client/src/test/java/org/uberfire/ext/widgets/core/client/editors/defaulteditor/UploadEventTest.java]: License please.

Comment (jhrcek) [reply uberfire-extensions/uberfire-widgets/uberfire-widgets-core/uberfire-widgets-core-client/src/test/java/org/uberfire/ext/widgets/core/client/editors/defaulteditor/UploadEventTest.java]: @tomasdavidorg avoid that kind of mistake in the future by configuring your IntelliJ using [these steps](https://github.com/kiegroup/droolsjbpm-build-bootstrap) (ctrl+f second occurence of "Set the correct license header")

Comment (tomasdavidorg) [reply uberfire-extensions/uberfire-widgets/uberfire-widgets-core/uberfire-widgets-core-client/src/test/java/org/uberfire/ext/widgets/core/client/editors/defaulteditor/UploadEventTest.java]: Fixed.

**Comment by ederign** [comment]
jenkins please retest this

**Pull request merged**